### PR TITLE
[IOCOM-1897] Remove Green Pass

### PR DESCRIPTION
**Pull request by Vangaorth**
## Short description
This PR removes the Green Pass mock implementation

## List of changes proposed in this pull request
- Relevant code has been removed
- One legacy message has been left in the message list, in order to test message selection for IO App

## How to test
Using the IO App, check that messages are still working.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -123,12 +123,12 @@ const defaultConfig: IoDevServerConfig = {
...
     withCTA: false,
-    withEUCovidCert: false,
+    generateLegacyGreenPassMessage: true,
     withValidDueDateCount: 0,
     withInValidDueDateCount: 0,
     // sending 2 messages at minimum to allow for basic pagination
-    standardMessageCount: 2,
-    archivedMessageCount: 1,
+    standardMessageCount: 0,
+    archivedMessageCount: 0,
```
Comment (Ladirico) [inline src/config.ts:131]: Why did you remove the payment messages?

Comment (Vangaorth) [reply src/config.ts]: My bad, reverted

**Review by Ladirico — APPROVED** [review]
LGTM! 🚀
